src/TRoomDB.cpp

```diff
@@ -252,21 +258,51 @@ TArea * TRoomDB::getArea( int id )
     }
 }
 
-void TRoomDB::setAreaName( int areaID, QString name )
+bool TRoomDB::setAreaName( int areaID, QString name )
 {
+    if( areaID < 1 ) {
+        qWarning( "TRoomDB::setAreaName((int)areaID, (QString)name): WARNING: Suspect areaID: %n supplied.", areaID );
```

Comment:
I think we may have got our wires crossed - when I put in a debug/warning message I try to put the class::method at the beginning so I (or any other person) can find where it came from - but for methods that share a name the argument types are needed for disambiguation - that is why this particular message has that extra detail.  I was agreeing with you that it would be nice if there was a #pragma or macro that could provide the detail auto-magically - it __wasn't__ an attempt to make more work for a future reviser. :open_mouth: 
Anyone who goes changing a method's signature or other details would be well advised to make sure that debug messages are kept up to date, I try to. That this is does not always happen is unfortunate but understood; I mean, look at the cases of (std::)cout that are still cropping up!